MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -862,6 +872,8 @@ namespace WPEFramework {
             else {
                 LOGWARN("Ignoring unexpected event - owner: %s, eventId: %d!!", owner, eventId);
             }
+          }
```

Comment:
Can we have a log statement here to indicate that abort flag has been set and hence ignoring the event ?

Reply:
Done